### PR TITLE
lookup: update flaky tags

### PR DESCRIPTION
**Pull request by gdams**
adding a few flaky tags, mainly on rhel

### CODE REVIEW

**Comment by gibfahn** [comment]
If we're going to mark things as flaky we should at least include the failing test output in the PR description. Otherwise if a module author wants to fix the flakyness they have nothing to go on.

File: lib/lookup.json [hunk 1]
```diff
@@ -72,7 +74,7 @@
   },
   "eslint": {
     "prefix": "v",
-    "flaky": ["ppc", "v7", "s390"],
+    "flaky": ["ppc", "v7", "s390", {"ubuntu": "16.04"}],
```
Comment (MylesBorins) [inline lib/lookup.json]: do we support this syntax for ubuntu? I don't recall seeing this before

Comment (gdams) [inline lib/lookup.json]: technically we should, i'll double check but we are obtaining the version number so I figured that putting ubuntu and the version number in an object would work

**Comment by gdams** [comment]
CI: https://ci.nodejs.org/view/Node.js-citgm/job/gibfahn-citgm-smoker-more-platforms/91/

**Pull request merged**